test/packages/other/number_formats/data_stream/numbers/_dev/test/pipeline/test-example.log-expected.json

```diff
@@ -57,7 +57,7 @@
         },
         {
             "example": {
-                "float": 6.9200717685635164E18,
+                "float": 6.920071768563516E18,
```

Comment:
It seems that something has changed on parsing of big numbers in Elasticsearch 8.5.0, let's hope this doesn't affect the tests of many integrations :crossed_fingers:

Reply:
Reported in https://github.com/elastic/elasticsearch/issues/91563 just in case.